server/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+python_files = *_features.py
+python_classes = *Features
+python_functions = should_*
```

Comment:
These three lines configure how pytest discovers tests.

- Look for files that end with `_features.py`
- Look for classes in those files that end with `_features.py`
- Look for functions inside those files and classes that start with `should_`